virt/virt-4-10-release-notes.adoc

```diff
@@ -164,6 +164,20 @@ To use the HPP to provide storage for virtual machine disks, configure it with d
 [id="virt-4-10-known-issues"]
 == Known issues
 
+* If you deploy the xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[hostpath provisioner] on a cluster where any node has a fully qualified domain name (FQDN) that exceeds 63 characters, the provisioner fails to bind PVCs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057157[*BZ#2057157*])
```

Comment:
It is actually worse than 63 characters, its only 42 characters anything longer will fail. This is because a label is created by prefixing `external-provisioner-` to the node name and turning that into a label.

Reply:
I will update it to 42 chars and then merge. Thanks!